.github/integration/tests/cancel/40_cancel_test.sh

```diff
@@ -0,0 +1,480 @@
+#!/bin/bash
+
+pip3 install s3cmd
+
+cd dev_utils || exit 1
+
+chmod 600 certs/client-key.pem
+
+C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | sed -e 's/.* //' -e 's/"//g')
+export C4GH_PASSPHRASE
+
+ENC_MD5=$(md5sum largefile.c4gh | cut -d' ' -f 1)
+ENC_SHA=$(sha256sum largefile.c4gh | cut -d' ' -f 1)
+
+## check that all containers are running
+docker ps -a
+
```

Comment:
Just for log output or should we react and bail out/wait if wrong?

Reply:
Just a sanity check, the compose file has dependencies so that the container running the test scripts should wait for inbox to be ready before it starts.